packages/plugins/typescript/enum-array/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@graphql-codegen/typescript-enum-array",
+  "version": "1.0.0",
```

Comment:
Should I down this version?

Reply:
Yeah, start with `0.0.0` so next major will be `1.0.0`. Thanks!